Services/CO.CDP.Person.WebApi/Api/Person.cs

```diff
@@ -106,6 +107,26 @@ await useCase.Execute(command)
                 operation.Responses["204"].Description = "Person deleted.";
                 return operation;
             });
+
+        app.MapPost("/persons/{personId}/associate",
```

Comment:
Shouldn't this be part of the Tenant Api? There seems to be a stub for it in there: `"/tenants/{tenantId}/assign-user`

Reply:
The naming there is confusing though (user vs person?) so it might need tweaking.